appveyor.yml

```diff
@@ -6,20 +6,20 @@ init:
 environment:
   matrix:
     - nodejs_version: '8'
-      webpack_version: latest
+      webpack_version: 3.8.1
       job_part: test
     - nodejs_version: '6'
-      webpack_version: latest
+      webpack_version: 3.8.1
       job_part: test
     - nodejs_version: '4.8'
-      webpack_version: latest
+      webpack_version: 3.8.1
       job_part: test
 build: 'off'
 matrix:
   fast_finish: true
 install:
   - ps: Install-Product node $env:nodejs_version x64
-  - npm i -g npm@latest
+  - npm i -g npm@^5.0.0
```

Comment:
Latest is better here

Reply:
Latest, i.e. v6 will fail with Node 4. latest npm uses es6.

Reply:
Had to drop support for node 4 & 6 and updated to latest